client/app/stories/Styleguide.stories.jsx

```diff
@@ -0,0 +1,358 @@
+import React, {Fragment} from 'react';
+import { storiesOf } from '@storybook/react';
+import css from '../styles/_global.scss';
+
+storiesOf('Styleguide', module)
+  .add('Buttons', () => (
+    <div
+      style={{
+        display: 'inline',
+        Right: '10px',
+        margin: '0 auto',
+        textAlign: 'center',
+        backgroundColor: 'transparent',
+        paddingBottom: '10px',
+      }}
+    >
+      <div
```

Comment:
Looks like this style exists in multiple places here, line 46, and line 75. 

We can do something like this:
```
const buttonsGroupStyle = {
  display: 'inline',
  right: '10px',
  margin: '0 auto',
  textAlign: 'center',
  backgroundColor: 'transparent',
  paddingBottom: '10px',
};
```

And then we can do `<div style={buttonsGroupStyle}>`